Student/Student/Init/FillDatabaseQuery.sql

```diff
@@ -0,0 +1,205 @@
+			@CurrentUserId,
+			GETDATE()
+		)
+
+	INSERT INTO [dbo].[Grade] ([StudentId], [GroupDisciplineId], [Grade], [StateGrade], [CreatedBy], [CreatedDateTime])
+	VALUES
+		(
+			(SELECT [StudentId] FROM [dbo].[Student] WHERE [FirstName] = N'Іщук'), 
+			(SELECT [GroupDisciplineId] FROM [GroupDiscipline] JOIN [AcademicDiscipline] ON [DisciplineId] = [AcademicDisciplineId] WHERE [DisciplineName] = N'Математика'),
+			5,
+			0,
+			@CurrentUserId,
+			GETDATE()
+		),
+		(
+			(SELECT [StudentId] FROM [dbo].[Student] WHERE [FirstName] = N'Іщук'), 
+			(SELECT [GroupDisciplineId] FROM [GroupDiscipline] JOIN [AcademicDiscipline] ON [DisciplineId] = [AcademicDisciplineId] WHERE [DisciplineName] = N'Математика'),
+			4,
+			0,
+			@CurrentUserId,
+			GETDATE()
+		),
+		(
+			(SELECT [StudentId] FROM [dbo].[Student] WHERE [FirstName] = N'Іщук'), 
+			(SELECT [GroupDisciplineId] FROM [GroupDiscipline] JOIN [AcademicDiscipline] ON [DisciplineId] = [AcademicDisciplineId] WHERE [DisciplineName] = N'Українська мова та література'),
+			5,
+			0,
+			@CurrentUserId,
+			GETDATE()
+		),
+		(
+			(SELECT [StudentId] FROM [dbo].[Student] WHERE [FirstName] = N'Тороканець'), 
+			(SELECT [GroupDisciplineId] FROM [GroupDiscipline] JOIN [AcademicDiscipline] ON [DisciplineId] = [AcademicDisciplineId] WHERE [DisciplineName] = N'Математика'),
+			3,
+			0,
+			@CurrentUserId,
+			GETDATE()
+		),
+		(
+			(SELECT [StudentId] FROM [dbo].[Student] WHERE [FirstName] = N'Тороканець'), 
+			(SELECT [GroupDisciplineId] FROM [GroupDiscipline] JOIN [AcademicDiscipline] ON [DisciplineId] = [AcademicDisciplineId] WHERE [DisciplineName] = N'Українська мова та література'),
+			2,
+			0,
+			@CurrentUserId,
+			GETDATE()
+		),
+		(
+			(SELECT [StudentId] FROM [dbo].[Student] WHERE [FirstName] = N'Тороканець'), 
+			(SELECT [GroupDisciplineId] FROM [GroupDiscipline] JOIN [AcademicDiscipline] ON [DisciplineId] = [AcademicDisciplineId] WHERE [DisciplineName] = N'Українська мова та література'),
+			4,
+			0,
+			@CurrentUserId,
+			GETDATE()
+		)
+
+	INSERT INTO [dbo].[StudentRating] ([StudentId], [StudentRating], [StipendSumm],  [CreatedBy], [CreatedDateTime])
+	VALUES 
+		((SELECT [StudentId] FROM [dbo].[Student] WHERE [FirstName] = N'Іщук'), 4.5, 1000, @CurrentUserId, GETDATE()),
+		((SELECT [StudentId] FROM [dbo].[Student] WHERE [FirstName] = N'Тороканець'), 4, 1000, @CurrentUserId, GETDATE()),
+		((SELECT [StudentId] FROM [dbo].[Student] WHERE [FirstName] = N'Браїлко'), 5, 1500, @CurrentUserId, GETDATE()),
+		((SELECT [StudentId] FROM [dbo].[Student] WHERE [FirstName] = N'Красицький'), 3, 1000, @CurrentUserId, GETDATE())
+
+	COMMIT TRANSACTION;
+END TRY
+BEGIN CATCH
+	ROLLBACK TRANSACTION;
+
+	SELECT 
+		ERROR_NUMBER() AS ErrorNumber, 
+		ERROR_SEVERITY() AS ErrorSeverity, 
+		ERROR_STATE() AS ErrorState, 
+		ERROR_PROCEDURE() AS ErrorProcedure, 
+		ERROR_LINE() AS ErrorLine, 
+		ERROR_MESSAGE() AS ErrorMessage;
+END CATCH
```

Comment:
```suggestion
END CATCH
GO
```
Don't forget to use the 'GO' operator